src/time_tool.cc

```diff
@@ -85,13 +87,15 @@ bool ParseCivilSpec(const std::string& args, cctz::civil_second* when) {
 
 const char* WeekDayName(cctz::weekday wd) {
   switch (wd) {
+    // clang-format off
```

Comment:
I don't have a problem with what clang-format does here, so I don't want to disable it.  I'm just choosing not to make the edits until we have a need to otherwise modify these lines.  That is, the whitespace churn isn't worth it in isolation.  So, leave things be for now, unless there is some other good reason to make an edit (like marking those sections we know we don't want to be reflowed).

Reply:
Okay. How do you propose to avoid formatting changes when editors/clang-format format the entire file as soon as one saves the file? The tools work at the file level. However, undoing these changes requires one to manually defeat clang-format, which appears to contradict the intention behind the guards and using `clang-format`? 

I'm not sure how defeating clang-format manually here helps maintain consistency. The idea was to format files so that any future feature-related changes would not have to include incidental formatting changes that were unrelated to the feature. Thoughts?

Reply:
> How do you propose to avoid formatting changes when editors/clang-format format the entire file as soon as one saves the file?

I don't propose to do anything except reject changes that make whitespace-only edits to code they are otherwise not touching.

> The idea was to format files so that any future feature-related changes would not have to include/manually manage incidental formatting changes that were unrelated to the feature.

I agree that those future changes should not include incidental formatting changes.  They should just format the new/changed code as they see fit, and I'm happy for that to be as clang-format chooses (except in the few circumstances we've already discussed, hence the need for some extra controls).

> All the files were formatted to reduce churn, imo.

I guess I disagree.  Formatting for formatting's sake is the source of churn.